check.sh

```diff
@@ -65,7 +65,7 @@ flake8 pippy spmd test/spmd
 
 # mypy spmd test/spmd
 echo; echo "Running mypy ..."
-mypy spmd pippy test examples
+mypy spmd pippy examples
```

Comment:
@kwen2501 after restricting more mypy linting with pytorch, there emerges tons of failures in tests, imo we shouldn't require strong type annotation in tests as that will slow down the dev efficiency, let me know if this not look good to you

Reply:
This might also stay true for examples? I am changing mypy to only run under `pippy` and `spmd`